packages/tsdl/tsdl.0.9.6/opam

```diff
@@ -22,6 +22,9 @@ depends: [
 build: [[
   "ocaml" "pkg/pkg.ml" "build"
            "--pinned" "%{pinned}%" ]]
+depexts: [
```

Comment:
I think we should add this to conf-sdl2 instead

Reply:
I didn't do this because currently this rule only works for SDL version <= 2.0.7 (because this is what cygwin has)
and the same happens to be true for tsdl.0.9.6.
I principle I agree it should go to conf-sdl2, but I don't know how to check the SDL version there.

Reply:
Ideally, each version of tsdl should have a requirement on the corresponding SDL version, but again I don't know if this is feasible. Maybe we could have several conf-sdl2 (for instance conf-sdl2.2.0.7, etc.)?